### PR TITLE
Build third party libpqxx via CMake and bump version

### PR DESCRIPTION
**Pull request by tobodner**
* Fix TODO to build libpqxx via CMake
* Rebase libpqxx submodule on origin/master
* Add postgresql-server-dev to Linux dependencies, which contains required header file pq_type.h since PostgreSQL 9.3
* Update install script
* Update dependencies file

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -51,7 +51,7 @@ if echo $REPLY | grep -E '^[Yy]$' > /dev/null; then
             echo "Installing dependencies (this may take a while)..."
             if sudo apt-get update >/dev/null; then
                 boostall=$(apt-cache search --names-only '^libboost1.[0-9]+-all-dev$' | sort | tail -n 1 | cut -f1 -d' ')
-                sudo apt-get install --no-install-recommends -y clang-7 libclang-7-dev clang-tidy-7 clang-format-7 gcovr python2.7 python-pexpect python-glob2 graphviz gcc-8 g++-8 llvm llvm-7-tools libnuma-dev libnuma1 libtbb-dev cmake libreadline-dev libncurses5-dev libsqlite3-dev parallel $boostall libpq-dev systemtap systemtap-sdt-dev autoconf &
+                sudo apt-get install --no-install-recommends -y clang-7 libclang-7-dev clang-tidy-7 clang-format-7 gcovr python2.7 python-pexpect python-glob2 graphviz gcc-8 g++-8 llvm llvm-7-tools libnuma-dev libnuma1 libtbb-dev cmake libreadline-dev libncurses5-dev libsqlite3-dev parallel $boostall libpq-dev postgresql-server-dev-all systemtap systemtap-sdt-dev autoconf &
```
Comment (mrks) [inline install.sh]: Packages added here should also be added to the Dockerfile. Maybe you could add a comment to that regard above this line

Comment (mrks) [inline install.sh]: What happens to OS X?

Comment (tobodner) [reply install.sh]: Good question. My understanding is that the split of header files (and related issues) is specific to Debian-based Linux distributions: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=314427
I will keep an eye on the macOS Jenkins job; I do not have a Mac and cannot test locally.

Comment (Bouncner) [reply install.sh]: https://www.apple.com/de/shop/buy-mac/imac-pro

Comment (tobodner) [reply install.sh]: Thanks for pointing this out. I sorted and synced the two package lists for consistency and convenience. I also added above comment.

**Comment by tobodner** [comment]
The Mac mini machine in the CI pipeline needs postgresql to be installed.

**Comment by tobodner** [comment]
postgresql has been installed and checks are passing.

**Comment by tobodner** [comment]
> That is brew installable?

Yes, please see the install script.
You can run brew install postgresql.

File: install.sh [hunk 2]
```diff
@@ -20,8 +20,8 @@ if echo $REPLY | grep -E '^[Yy]$' > /dev/null; then
         echo "Installing dependencies (this may take a while)..."
         if brew update >/dev/null; then
             # python2.7 is preinstalled on macOS
-            # check, for each programme individually with brew, whether it is already installed
-            # due to brew issues on MacOS after system upgrade
+            # check, for each program (aka. formula) individually with brew, whether it is already installed due to brew issues on MacOS after system upgrade
+            # NOTE: The Mac CI server does not execute the install.sh - formulas need to be installed manually.
```
Comment (Bouncner) [inline install.sh]: Ahhh ... thanks. That caused my confusion yesterday.

**Pull request merged**